jdaviz/core/template_mixin.py

```diff
@@ -2970,7 +2970,8 @@ def _get_continuum(self, dataset, spectral_subset, update_marks=False, per_pixel
         if per_pixel:
             if self.app.config != 'cubeviz':
                 raise ValueError("per-pixel only supported for cubeviz")
-            full_spectrum = self.app._jdaviz_helper.get_data(self.dataset.selected)
+            full_spectrum = self.app._jdaviz_helper.get_data(self.dataset.selected,
+                                                             use_display_units=True)
```

Comment:
I feel like @cshanahan1 had just removed this.  Was there a reason for that or did it just not seem to affect that use-case (see https://github.com/spacetelescope/jdaviz/pull/3211/files#r1787615549)